src/sentry/static/sentry/app/views/eventsV2/queryList.tsx

```diff
@@ -142,7 +142,7 @@ class QueryList extends React.Component<Props> {
           key={`${index}-${eventView.id}`}
           to={to}
           title={eventView.name}
-          subtitle={t('Saved Query')}
+          subtitle={t('Last ') + `${eventView.statsPeriod}`}
```

Comment:
@doralchan `eventView.statsPeriod` may not be provided in some cases; instead `eventView.start` and `eventView.end` are likely provided since absolute datetime range could be saved when chosen through the global selection header.

Would it be helpful if `eventView` has provided some method to generate relative dates or date ranges? 
We could have a method using `moment`: https://momentjs.com/docs/#/displaying/

Reply:
In this case, I just wanted the cards to match the global header timeline display. Is a new method required to achieve that?

Reply:
@doralchan the new method was a suggestion.

But there are `eventView.statsPeriod`, `eventView.start`, and `eventView.end` at your disposal. It might be a good idea to add more code here to help you achieve what you want. Just remember, that some of these variables may be `undefined`, which is something we want to account for. 


------

To help get you started, I've used `getUtcToLocalDateObject()` function here to convert `start` and `end` to `Date` objects: https://github.com/getsentry/sentry/blob/f542b0291042ef449ddca80c14b27328cf4334ce/src/sentry/static/sentry/app/views/eventsV2/miniGraph.tsx#L42-L45

Thee `moment` library can help you format the datetime strings.

Reply:
Got it - I added something